pkg/types/defaults.go

```diff
@@ -9,10 +9,27 @@ const (
 )
 
 var (
-	NcpSections      = []string{"DEFAULT", "ha", "k8s", "coe", "nsx_v3", "vc"}
-	AgentSections    = []string{"DEFAULT", "k8s", "coe", "nsx_node_agent", "nsx_kube_proxy"}
-	OperatorSections = []string{"DEFAULT", "ha", "k8s", "coe", "nsx_v3", "vc", "nsx_node_agent", "nsx_kube_proxy"}
-	BootstrapOptions = []string{"DEFAULT", "nsx_node_agent"}
+	NcpSections                = []string{"DEFAULT", "ha", "k8s", "coe", "nsx_v3", "vc"}
+	AgentSections              = []string{"DEFAULT", "k8s", "coe", "nsx_node_agent", "nsx_kube_proxy"}
+	OperatorSections           = []string{"DEFAULT", "ha", "k8s", "coe", "nsx_v3", "vc", "nsx_node_agent", "nsx_kube_proxy"}
+	AgentRestartSections       = []string{"DEFAULT", "nsx_node_agent", "nsx_kube_proxy"}
+	BootstrapRestartOptionKeys = map[string][]string{
+		"DEFAULT": {
+			"log_dir", "log_file", "log_rotation_file_max_mb", "log_rotation_backup_count",
+		},
+		"nsx_node_agent": {
+			"enable_ipv6", "use_nsx_ovs_kernel_module", "ovs_db_sock", "ovs_bridge", "ovs_uplink_port", "mtu",
+		},
+		"coe": {
```

Comment:
why we add "coe":{node_type} for BootstrapRestartOptionKeys since we see previous BootstrapOptions only have DEFAULT and nsx_node_agent section?

Reply:
Adding this because it's used in `init_k8s_node` in bootstrap, so I thinks we need add this field unless the field is not allowed to be updated.

> why we add "coe":{node_type} for BootstrapRestartOptionKeys since we see previous BootstrapOptions only have DEFAULT and nsx_node_agent section?

Reply:
I noted that "coe":{node_type} is an advanced option, which means is not visible to users. 

cfg.StrOpt('node_type',
               choices=[constants.HOSTVM_NODE_TYPE, constants.WCP_WORKER],
               default=constants.HOSTVM_NODE_TYPE,
               advanced=True,
               help='The type of container host node'),
               
we only run once   init_k8s_node in  bootstrap, during start, after then, init_container exit.    

So，I think users don't need to update this option after running.